config/locales/en.yml

```diff
@@ -443,6 +443,7 @@ en:
           incorrect_format: The selected file must be formatted correctly
           empty: The selected file must contain some data
           format: "%{message}"
+          virus_found: Virus Found
```

Comment:
i think if mark hasn't responded yet we should use the same message that is in the Runner for now

Reply:
All good, he's responded and we've made the change 🚀